pkgs/profpatsch/nman/nman.1

```diff
@@ -0,0 +1,194 @@
+.Dd $Mdocdate$
+.Dt NMAN 1
+.Os
+.Sh NAME
+.Nm nman
+.Nd nix-shell for man pages
+.Sh SYNOPSIS
+.Nm
+.Op Fl h
+.Ar ATTR
+.Op Ar PAGE | SECTION Op PAGE
+.Sh DESCRIPTION
+.Nm
+is a simple tool allowing to open man pages of packages
+which are not installed using the power of the
+.Xr nix 1
+package manager.
+It allows you to specify a package and a man
+page and opens it directly using
+.Xr man 1 ,
+essentially implementing what you would expect
+.Ql nix-shell -p foo --run 'man foo'
+to do.
+.Pp
+You can invoke
+.Nm
+in three different ways:
+.Bl -ohang -offset 2n
+.It Nm Ar ATTR SECTION PAGE
+Evaluates the attribute
+.Ar ATTR
+of
+.Ql <nixpkgs>
+and searches all its outputs for a man page of name
+.Ar PAGE
+in man section
+.Ar SECTION .
+.Ar PAGE
+and
+.Ar SECTION
+work just like with
+.Xr man 1 .
+.It Nm Ar ATTR SECTION
+Like above, but
+.Ar ATTR
+is also used as the name of the man page.
+Note that
+.Ar SECTION
+must be a number or
+.Ql 3p .
+Otherwise the next invocation is assumed.
+.It Nm Ar ATTR PAGE
+Opens the man page
+.Ar PAGE
+from package
+.Ar ATTR .
+If multiple matches are found, pages from lower
+man sections are preferred, for example
+.Ql nman lowdown lowdown
+opens
+.Xr lowdown 1
+and not
+.Xr lowdown 3 .
+.It Nm Ar ATTR
+Uses the described method to find the man page
+.Ar ATTR
+from package
+.Ar ATTR .
+.El
+.Sh IMPLEMENTATION NOTES
```

Comment:
Let’s add a `DISCLAIMER` section before this one, to say something like “this tool is intended to be used interactively, since the parsing heuristic is implemented to do “the right thing”, not to be integrated in scripts or other programs”.

Reply:
Done in aaa0f437736023abfa34a9e95789c404f90b0d99